### DISCUSSION TITLE
Can anyone suggest an image library program to access Photoshop libraries?

### DISCUSSION THREAD

**Discussion by williamdrescher** [ImageMagick/ImageMagick#3835] [Digital Image Processing]
I have a large catalog of images in photoshop.  I am moving to ImageMagic.  
Can anyone suggest an image library program to access Photoshop libraries that will display images and allow selection by photoshop tags ?

**Comment by williamdrescher** [comment]
I forgot: environment: Win10

**Comment by urban-warrior** [comment]
Try Gimp.
